seed.lua

```diff
@@ -0,0 +1,22 @@
+function HandleSeedCommand( Split, Player )
+	Player:SendMessageInfo("Seed: " .. Player:GetWorld():GetSeed())
+	return true
+end
+
+
+
+
+
+function HandleConsoleSeed( Split )
+	if (#Split == 2) then
```

Comment:
Lua Performance Tip (tm)(c)(r): Instead of counting all elements, check that the index you want to access:
```lua
if (not(Split[2])) then
```
makes one table access and one negation. Fast

```lua
if (#Split == 2) then
```
iterates over all elements in the `Split` table, then compares to the number.